src/main/java/org/elasticsearch/action/admin/indices/seal/SealIndicesResponse.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.action.admin.indices.seal;
+
+import org.elasticsearch.action.ActionResponse;
+import org.elasticsearch.cluster.routing.ShardRouting;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.ToXContent;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.indices.SyncedFlushService;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * A response to a seal action on several indices.
+ */
+public class SealIndicesResponse extends ActionResponse implements ToXContent {
+
+    private Set<SyncedFlushService.SyncedFlushResult> results;
```

Comment:
can we make it final and assign it in the default ctor... in readFrom we can just call `results.clear()`